src/content/docs/apm/agents/go-agent/configuration/go-agent-configuration.mdx

```diff
@@ -595,6 +595,8 @@ In this and the following examples, `config` represents your New Relic config st
   </Collapser>
 </CollapserGroup>
 
+If you're using New Relic APM and CodeStream, see [how to associate repositories](https://docs.newrelic.com/docs/codestream/start-here/codestream-new-relic/#repo-url) and [how to associate build SHAs or release tags] with errors inbox.
```

Comment:
missing link for [how to associate build SHAs or release tags]

Reply:
Actually, it looks like each file in this PR is updated with the same content and is missing this link?

Reply:
Fixed all of these. Thanks!